src/Database/Schema/Migrations/Filesystem.hs

```diff
@@ -104,34 +123,34 @@ migrationFromPath path = do
             Just m -> return m
         _ -> throwFS $ "Error in " ++ (show path) ++ ": missing required field(s): " ++ (show missing)
 
-getFields :: YamlLight -> [(String, String)]
+getFields :: YamlLight -> [(Text, Text)]
 getFields (YMap mp) = map toPair $ Map.assocs mp
     where
-      toPair (YStr k, YStr v) = (unpack k, unpack v)
+      toPair (YStr k, YStr v) = (T.decodeUtf8 k, T.decodeUtf8 v)
```

Comment:
Due to this change, can you also update `MOO.TXT` to mention that migrations are assumed to be UTF-8 encoded?